internal/cephfs/core/fsjournal.go

```diff
@@ -124,6 +124,11 @@ func CheckVolExists(ctx context.Context,
 			return nil, cerrors.ErrClonePending
 		}
 		if cloneState == cephFSCloneFailed {
```

Comment:
Will be better if we log the `cloneState` outside this block of code?

Reply:
it might not be helpful as the clone takes time to complete with that we will see more in-progress logs. and we are already returning pending state to the user and it's logged as GRPC error.